unstructured/staging/base.py

```diff
@@ -228,17 +233,78 @@ def convert_to_csv(elements: List[Element]) -> str:
 
 
 @requires_dependencies(["pandas"])
-def convert_to_dataframe(elements: List[Element], drop_empty_cols: bool = True) -> "pd.DataFrame":
+def get_default_dtypes() -> dict:
```

Comment:
```suggestion
def get_pandas_default_dtypes() -> dict:
```
maybe? had no idea what a dtype was until I looked inside this function